cmd/podman/images/mount.go

```diff
@@ -0,0 +1,126 @@
+package images
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/containers/libpod/v2/cmd/podman/registry"
+	"github.com/containers/libpod/v2/cmd/podman/utils"
+	"github.com/containers/libpod/v2/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	mountDescription = `podman image mount
+    Lists all mounted images mount points if no images is specified
+
+  podman image mount IMAGES-NAME-OR-ID
+    Mounts the specified image and outputs the mountpoint
+`
+
+	mountCommand = &cobra.Command{
+		Use:   "mount [flags] [IMAGES]",
+		Short: "Mount an images's root filesystem",
+		Long:  mountDescription,
+		RunE:  mount,
+		Example: `podman image mount ctrID
+  podman image mount ctrID1 ctrID2 ctrID3
+  podman image mount --all`,
+	}
+)
+
+var (
+	mountOpts entities.ImageMountOptions
+)
+
+func mountFlags(flags *pflag.FlagSet) {
+	flags.BoolVarP(&mountOpts.All, "all", "a", false, "Mount all imagess")
+	flags.StringVar(&mountOpts.Format, "format", "", "Change the output format to Go template")
```

Comment:
Maybe I'm missing something, but currently this option only accepts `json` argument, not Go template format string.

Reply:
Man pages below also agrees it's just json.

Reply:
Cut and paste from cmd/podman/containers/mount.go.

Fixed in both places.